plugins/disk/quota2percent_

```diff
@@ -0,0 +1,259 @@
+     exit 128
+  fi
+
+###################################################
+# Data reading sections                           #
+###################################################
+
+# Reading the quotes for the selected device, using repquota
+  if repquota "/dev/$Id" &> /dev/null; then     
+     readarray Quotas < <( repquota "/dev/$Id" | grep " -- " )
+  else
+     echo "No limitations administered via 'quota' for $Id" >&2
+     # If no limitatitons administered: create a dummy
+     Quotas[0]="root -- 1 1 1 1 1 1"
+     Low_UID="never"
+  fi
+
+  readarray Totals < <( df "/dev/$Id" )
+
+# Get the count of Users 
+   Users=${#Quotas[@]}
+
+
+###################################################
+# Munin Configuration Section                     #
+###################################################
+
+  if [ "$1" = "config" ]; then
+
+     # Localisation of the graphic texts 
+     case $Language in
+          de)
+            echo "graph_title    Quota-Hard-Limit von $Id"
+            echo "graph_vlabel   Nutzung in % Hardlimit"
+            echo "graph_info     Die Grafik zeigt die Belegung des durch Quota reglementierten Speicherplatzes f�r alle regul�ren Nutzer (UID >= $Min_UID) in Prozent des Hardlimits."
+            Total_txt="Su. aller Nutzer"
+            Total_info="Inklusive Systemnutzer (UID < $Min_UID)"
+            ;;
+          es) 
+            echo "graph_title    Cuota de l�mite absoluto de $Id" 
+            echo "graph_vlabel   el % de uso del l�mite duro" 
+            echo "graph_info     El gr�fico muestra la disponibilidad de espacio regulado por cuotas para todos los usuarios regulares (UID >= $Min_UID) como porcentaje de l�mites duros."
+            Total_txt="Suma de todos los usuarios "
+            Total_info="La inclusi�n de usuario del sistema (UID < $Min_UID) "
+            ;;
+           *)
+            echo "graph_title    quota hard limit of $Id"
+            echo "graph_vlabel   Usage in %"
+            echo "graph_info     The graphic shows the allocation of the quota-regulated storage space for all regular users (UID >= $Min_UID) as a percentage of the hard limit ."
+            Total_txt="all users"
+            Total_info="system users (UID < $Min_UID) included" 
+            ;;
+     esac
+
+     # Defaults configuration 
+       echo "graph_category disk"
+       echo "graph_args     --lower-limit 0 --upper-limit 100"
+       echo "graph_printf   %5.2lf %%"
+       echo "graph_scale    no"
+
+     # Processing the individual user
+       for((i=0; i<"$Users"; i++));do
+         Quota=( ${Quotas[$i]} )
+         User=${Quota[0]}
+       # solve the root problem
+         Fieldname="$(clean_fieldname "$User")"
+
+       # Determine the currently processing UID
+         Cur_UID="$(id -u "$User")"
+
+       # skip if actual user a system user und low_uid ist set to never  
+         if [ ! "$Low_UID" == "never" ] || [ ! "$Cur_UID" -lt "$Min_UID" ]; then
```

Comment:
This logic operation is a bit hard to read.
Maybe `[ "$Low_UID" != "never" ]` would be easier to read than `[ ! "$Low_UID" == "never" ]`?
Same for `! ... -lt` - you could write it as `-ge` instead. Negation causes brain pain :)
I guess that you tried to express something like `not (A and B)` -> `(not A) or (not B)`. But this takes quite a bit of time to understand for the next reader, I think.

In general I would suggest to just write down your human description ("skip if actual user a system user und low_uid ist set to never"):
```
[ "$Cur_UID" -lt "$Min_UID" ] && [ "$Low_UID" = "never" ] && continue
```